src/components/Tables/DomainTable/DomainTable.tsx

```diff
@@ -190,11 +180,20 @@ const DomainTable: React.FC<DomainTableProps> = ({
 
 				completed++;
 
+				if (completed === domains.length) {
+					setIsSubscribed(false);
```

Comment:
In this case, the `isSubscribed` variable should *not* be a React state variable. It is a local variable only to the Effect, and should be defined above. (i will mark it)

Make sure you look at the example code I linked earlier; https://github.com/zer0-os/zNS-dapp/blob/c685f0d795ec31c3d3eb66506aeff53d210f2250/src/components/Profile/Profile.tsx#L86

In the example, isSubscribed is not a state variable, it shouldn't be one here either.